### ISSUE TITLE
favicon new mail count broken

### ISSUE THREAD

**Issue by graxlop** [RainLoop/rainloop-webmail#1385]
After updating from version 1.10.5.192 to 1.11.0.203, the red indicator with the number of new e-mails has disappeared from the favicon.

**Version 1.10.5.192:**
![rainold](https://cloud.githubusercontent.com/assets/17195442/24479755/bde9eada-14ae-11e7-8339-56803f507e3c.png)

**Version 1.11.0.203:**
![rainnew](https://cloud.githubusercontent.com/assets/17195442/24479766/d1de85a0-14ae-11e7-8a1f-e40ebe9eeeec.png)

**Comment by RainLoop** [comment]
Yes, unfortunately browsers do not work well with dynamic favicons, they cache them and as a result it only confuses users when they see obsolete data on the site icon.

That's why I decided to remove this functionality from RainLoop.
